package.json

```diff
@@ -22,6 +22,104 @@
+      "properties": {
+        "SolidityArguments": {
+          "markdownDescription": "A list of arguments to pass to the Solidity compiler.\nThe arguments will be passed as is, without any formatting, in the same order.\n\n**Example:**\n\n```js\n['--optimize', '--optimize-runs', '200']\n```",
+          "type": "string",
+          "pattern": "^(\\[(\\'[0-9a-z\\-]+\\',(\\s?))*((\\'[0-9a-z\\-]+\\')*)\\])*$",
+          "patternErrorMessage": "Input must be an array of string commands",
+          "order": 0
+        },
+        "Solidity.DefaultDirectoryForPackagesDependencies": {
+          "description": "i.e: 'node_modules', 'lib'. This is used to avoid typing imports with that path prefix",
+          "type": "string",
+          "default": "node_modules",
+          "order": 1
+        },
+        "SolidityPackageDirectories": {
+          "markdownDescription": "For each package, gets the path to a directory including that Solidity package.\n\n**When to use it?**\n\nBy default we look for the packages in default directory. Set here if there are packages are in several different directory.\n\nValue should be a mapping between a package name and the path to the directory the package is at.\n\n**Example:**\n```\n{'package.name': '.../path'}```",
+          "type": "object",
+          "additionalProperties": "string",
+          "properties": {},
+          "order": 2
+        }
+      }
+    },
+    {
+    "title": "Prover defaults",
+    "order": 2,
+      "properties": {
+        "OptimisticLoop": {
+          "markdownDescription": "Ignore some executions of loop. [Read More](https://docs.certora.com/en/latest/docs/ref-manual/cli/options.html?highlight=--package#optimistic-loop)",
+          "type": "boolean",
+          "default": false,
+          "order": 0
+        },
+        "LoopUnroll": {
+          "description": "Loop Unroll expanation",
+          "type": "integer",
+          "exclusiveMinimum": -1,
+          "exclusiveMaximum": 10,
+          "default": 1,
+          "order": 1
+        },
+        "Duration": {
+          "description": "Maximal execution time per rule (in seconds)",
+          "type": "integer",
+          "exclusiveMinimum": -1,
+          "default": 600,
+          "order": 2
+        }
+      }
+  },
+  {
+    "title": "Prover Advance Settings",
+    "order": 3,
+    "properties": {
+      "AdditionalProperties": {
+        "markdownDescription": "[Additional properties explanation](https://docs.certora.com/en/latest/docs/ref-manual/cli/options.html?highlight=option)\n\n**Example:**\n\n```{'--property': 'someValue'}```",
+        "type": "object",
+        "order": 0,
+        "additionalProperties": "string",
+        "properties": {}
+      },
+      "LocalTypeChecking": {
+        "description": "Turning this off implies syntax error are reported after submitting job to server",
+        "type": "boolean",
+        "default": true,
+        "order": 1
+      },
+      "Staging": {
```

Comment:
It should be separated into two different inputs. You can see how it's defined in the current conf creation webview.
`Staging` is a boolean, and then there is another text input for the branch name.

Reply:
Oh, I see that it has a different format in the mockup.
Again, please consult with Nurit.

Reply:
So in this part I consulted Uri about how it should work, and he said that the default value should be empty/`production`, empty seemed less confusing and Nurit agreed.
In the current `create conf file` form, the default value was `staging: true, branch: master` which is wrong (this is the values for the certora employees not to the users). I changed it to the correct default value for both the form and the vscode settings, with  is `staging: false`. 
I also connected the `staging` value from the vscode settings in a way that if the value is empty, the form will show `staging: false` and if it has a value `value` the form will show `staging: true, brach: value`.